2016/20161108__vt__general__precinct.csv

```diff
@@ -10702,281 +10702,281 @@ Washington,President,,Woodbury,,Hillary Clinton. Tim Kaine,Democratic,271,82048
 Bennington,President,,Woodford,,Hillary Clinton. Tim Kaine,Democratic,94,82048
 Windsor,President,,Woodstock,,Hillary Clinton. Tim Kaine,Democratic,1319,82048
 Washington,President,,Worcester,,Hillary Clinton. Tim Kaine,Democratic,374,82048
-Addison,President,,Addison,,Donald Trump. Michael R ,Republican,337,82048

```

Comment:
It looks like "Pence" is missing from the candidate name.  I'll fix these in an upcoming pull request.